lib/plugin_version.rb

```diff
@@ -65,14 +65,14 @@ def request_params
     {}.tap do |params|
       params["id"] = @id unless @id.nil?
       params["access_token"] = @access_token
-      params["plugin_version[name]"] = @manifest["name"]
-      params["plugin_version[identifier]"] = @manifest["identifier"]
+      params["plugin_version[name]"] = @name
+      params["plugin_version[identifier]"] = @identifier
       params["plugin_version[category]"] = @manifest["type"]
       params["plugin_version[plugin_id]"] = @plugin.id
       params["plugin_version[manifest]"] = @manifest.to_json
       params["plugin_version[author_email]"] = @manifest["author_email"]
       params["plugin_version[version]"] = @manifest["manifest_version"]
-      params["plugin_version[platform]"] = @manifest["platform"]
+      params["plugin_version[platform]"] = @manifest["platform"] || nil
```

Comment:
why we need `||` nil? if its empty it will put nil

Reply:
you’re right. But I want here is to avoid the scenario where there is an empty string, since I know expect it to be nil in zapp. But i’ll wrap it differently to handle the case I want to handle more properly